lib/shipit/line_buffer.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Shipit
+  class LineBuffer
+    SEPARATOR = "\n"
+
+    def initialize(value = "")
+      @value = value
```

Comment:
maybe `@buffer` would be more telling than `@value`?

Reply:
I had that originally, but the class is a buffer, so it having a buffer value didn't feel right.

Reply:
It's very common though. Up to you, but I generally find names like "value" or "data" very undescriptive.

Reply:
The problem is it's not a `@buffer`, it's a `String` that holds the current value of a `LineBuffer`, so it's very difficult to name. `value` felt better than `string` or `buffer` or `data`, but it still doesn't feel right.

Reply:
It's a queue! That's it. I’m going with `@queue`.

Reply:
Thanks for picking up on this @casperisfine. 🙏  `@queue` sparks joy.